package.json

```diff
@@ -33,6 +33,8 @@
     "prettier_target": "*.{js,jsx,md,json} src/**/*.{js,jsx,md,json} test/**/*.{js,jsx,md,json} docs/**/*.{js,jsx,md,json} stories/**/*.{js,jsx,md,json} cypress/**/*.{js,jsx,md,json}"
   },
   "scripts": {
+    "test:accessibility": "lighthouse http://localhost:9002/iframe.html?id=single-vertical-list--basic --chrome-flags='--headless' --output=json --output-path=./test-reports/lighthouse/a11y-audit.json && yarn test:accessibility:report",
+    "test:accessibility:report": "node a11y-audit-parse.js",
```

Comment:
does this need to be a separate step?

Reply:
At first I was thinking I might have `test:accessibility` and `test:accessibility:ci` and then they could share that common step but now there is only a need for `test:accessibility` so I'll consolidate and just have the one NPM script for this.

Reply:
Done; just a single step now.